### DISCUSSION TITLE
Archiver tool command failing to connect to snowflake db

### DISCUSSION THREAD

**Discussion by alexwillingham98** [thoughtspot/cs_tools#173] [General]
This is another one of those that was working without me making any changes to my code and now it's failing. This command is run  on a weekly job that spins up a vm and gets the latest version of cs_tools. 

Command:.\cs_tools.exe tools archiver remove --tag 'INACTIVE' --syncer snowflake://$(Build.SourcesDirectory)/cs_tools/config/snowflake.cs_tools.upsert.toml --no-prompt --export-only --config main

Attached error to file.
[cs_toolserror.txt](https://github.com/user-attachments/files/17101570/cs_toolserror.txt)

**Comment by boonhapus** [comment]
> Could not connect to Snowflake backend after 2 attempt(s). Aborting

@alexwillingham98 This appears to be an issue with Snowflake, can you check the details in your `config/snowflake.cs_tools.upsert.toml` and see if they're correct?

**Reply by alexwillingham98** [reply]
Yeah I have, the weird thing is, I actually use that same file for another cs_tools command and it seems to work fine.

**Reply by boonhapus** [reply]
@alexwillingham98 Is it the same VM / startup script for each cs_tools command? I'm trying to think of what could be different between the working and non-working version. Archiver isn't doing anything special with the Snowflake syncer here.
